media/js/mozorg/about-leadership.js

```diff
@@ -37,9 +37,19 @@ if (typeof window.Mozilla === 'undefined') {
         });
     }
 
+    function getHash() {
+        var hash = window.location.hash;
+        if (hash.indexOf('#') > -1) {
+            hash = hash.split('#')[1];
+        }
+
+        return hash;
+    }
+
     // trigger modal on page load if hash is present and matches a person with a bio
     if (window.location.hash) {
-        var target = document.getElementById(window.location.hash.substr(1));
```

Comment:
It seems this broke semi-recently because `substr` is deprecated. Likely still works in older browsers but doesn't work in current Nightly. I swiped the getHash function from /products/vpn/ and I'll work on another PR to fix a few more uses of `hash.substr(1)` elsewhere.